libraries/AP_NavEKF3/AP_NavEKF3_core.h

```diff
@@ -935,6 +946,11 @@ class NavEKF3_core : public NavEKF_core_common
     // returns false if unsuccessful
     bool EKFGSF_resetMainFilterYaw();
 
+	// Fusion of body frame X and Y axis drag specific forces for multi-rotor wind estimation
```

Comment:
nitpick: extra space at the beginning of this line

Reply:
Fixed.